src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountTests.fs

```diff
@@ -0,0 +1,115 @@
+module BankAccountTests
+
+open System
+open Deciders.BankAccount
+open Deciders.BankAccountPrimitives
+open FsCheck
+open Ogooreck.BusinessLogic
+open Deciders.BankAccountDecider
+open FsCheck.Xunit
+
+let random = Random()
+let now = DateTimeOffset.UtcNow
+
+let spec =
+    Specification.For(decide, evolve, (fun () -> NotInitialised))
+
+let BankAccountOpenedWith =
```

Comment:
this syntax has better intellisense as you are not returning a lambda (that syntax does have its plce, but its not a random good thing to do)
```suggestion
let BankAccountOpenedWith
          bankAccountId now version =
```

Reply:
Good point, updated in https://github.com/oskardudycz/Ogooreck/pull/13/commits/9c3bd4fa7efd66b75bf111513ba31eea4892e0cb.